src/main/scala/com/typesafe/sbt/ProguardKeys.scala

```diff
@@ -58,6 +58,12 @@ trait ProguardKeys {
         |    public static void main(java.lang.String[]);
         |}""".stripMargin.format(name)
     }
+
+    def keepMethods(className: String, returnType: String, inputType: String, methodName: String): String = {
```

Comment:
Should it be called `keepStaticMethod`, since it only applies to static methods and a singe method name?

Reply:
Well I was thinking you could put `*`s in there if you want to keep methods that have part of a specific string. That was actually my personal use case, I wanted every method that started with 'abc'. But maybe `keepStaticMethod` would be a better name.